src/routes/profile/data.ts

```diff
@@ -27,13 +27,34 @@ export async function insertProfile(database: D1Database, { email, name, passwor
 		),
 		// TODO: insert into links table
 		database.prepare(`
-			INSERT INTO ${DBTables.PASSWORD} (
-				id, schemaVersion, password
+			INSERT INTO ${DBTables.ACCESS} (
+				id, schemaVersion, access_level, password, email
 			)
 			VALUES (
-				?, ?, ?
+				?, ?, ?, ?, ?
 			)
-		`).bind(id, schemaVersion, password)
+		`).bind(id, schemaVersion, 'volunteer', password, email),
+
+		database.prepare(`
+			INSERT INTO ${DBTables.ROLE} (
+				id, schemaVersion, happenedAt, insertedAt,
+				role, description, teamId, profileId
+			)
+			VALUES (
+				?, ?, ?, ?,
+				?, ?, ?, ?
+			)
+		`).bind(
+			id,
+			schemaVersion,
+			happenedAt,
+			insertedAt,
+			'volunteer',
+			'',
+			'',
+			id
+		)
+		// TODO : set TorontoJS team id 👆
```

Comment:
Just generate a random UUID and save it on `constants.ts`, we don't need to make a big deal out of it 😅